nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -357,9 +365,10 @@ in {
           else "localhost";
         defaultText = "localhost";
         description = lib.mdDoc ''
-          Database host or socket path. Defaults to the correct unix socket
-          instead if `services.nextcloud.database.createLocally` is true and
-          `services.nextcloud.config.dbtype` is either `pgsql` or `mysql`.
+          Database host or socket path.
+          If `services.nextcloud.database.createLocally` is true and
```

Comment:
I guess it might also be nice to reference options via like `[](#opt-services.nextcloud.database.createLocally)` then you can directly jump to the option in the manual.

Reply:
Done